Makefile

```diff
@@ -29,7 +29,7 @@ help:
 # http://marmelab.com/blog/2016/02/29/auto-documented-makefile.html
 	@grep -E '^[a-zA-Z0-9_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
 
-clean: ## Make a clean source tree
+clean-elyra:
```

Comment:
I don't see why `client` was split into `clean-elyra` and `uninstall` when neither of these is used elsewhere.

I also don't like the `-elyra` in the name and, if these do need to be split, would prefer to keep them as separate targets and create a combined target like `purge` or something as the helper.

Reply:
this was a request of @lresende in #545 I'm ok putting them back together

Reply:
I can use `purge` instead if @lresende still wants them separate

Reply:
To me clean will remove/delete stuff from the build env or modified by the build. The name, as I understand, has a de-facto standard as `clean`. Now, in the original case, we were using `clean` to also do unlink which before was being done from `install` as in JupyterLab 1.x same version extensions/packages were not being installed unless they were unlinked/uninstalled first, thus it was being forced during install. In case we still need to unlink before it can be updated, we need to make sure we perform that when `install` is called without `clean`.